Source/Core/Core/IOS/DolphinDevice.cpp

```diff
@@ -222,12 +223,29 @@ IPCReply DolphinDevice::GetSystemTime(const IOCtlVRequest& request) const
   // This ioctl is used by emulated software to judge if emulation is running too fast or slow.
   // By using Common::Timer, the same clock Dolphin uses internally for the same task is exposed.
   // Return elapsed time instead of current timestamp to make buggy emulated code less likely to
-  // have issuses.
+  // have issues.
   const u32 milliseconds = static_cast<u32>(m_timer.ElapsedMs());
   Memory::Write_U32(milliseconds, request.io_vectors[0].address);
   return IPCReply(IPC_SUCCESS);
 }
 
+IPCReply DolphinDevice::GetSystemTime(const IOCtlVRequest& request) const
+{
+  if (!request.HasNumberOfValidVectors(0, 1))
+  {
+    return IPCReply(IPC_EINVAL);
+  }
+
+  if (request.io_vectors[0].size != 8)
+  {
+    return IPCReply(IPC_EINVAL);
+  }
+
+  const u64 milliseconds = Common::Timer::NowMs();
```

Comment:
`Common::Timer::NowMs` doesn't necessarily return unix time. It's currently using `std::chrono::steady_clock`'s `time_since_epoch()` which is a non-specific epoch. If it's working, that's not guaranteed.

Reply:
Resolved in https://github.com/dolphin-emu/dolphin/pull/11269/commits/37888687c569080d4fa66928273de84b4ae01aec.